src/main/flight/pid.c

```diff
@@ -404,16 +409,17 @@ void pidInitFilters(const pidProfile_t *pidProfile)
         }
     }
 #endif
-#if defined(USE_D_CUT)
-    if (pidProfile->dterm_cut_percent == 0) {
-        dtermCutRangeApplyFn = nullFilterApply;
-        dtermCutLowpassApplyFn = nullFilterApply;
-    } else {
-        dtermCutRangeApplyFn = (filterApplyFnPtr)biquadFilterApply;
```

Comment:
@ctzsnooze the apply function pointers aren't being set in the new code

Reply:
dumb as f**k I can be - thanks